app/controllers/dashboard/cards_controller.rb

```diff
@@ -43,6 +45,34 @@ def get_flickr_images
                                                  'url_sq, url_m', 10, 1)
   end
 
+  def parsing_html
+    url = parsing_html_params[:url]
+    css_original = parsing_html_params[:css_original]
+    css_translated = parsing_html_params[:css_translated]
+
+    html = Nokogiri::HTML(open(url))
+    original_texts = html.css(css_original)
+    translated_texts = html.css(css_translated)
+
+    if original_texts.size == translated_texts.size && original_texts.size != 0
+      if original_texts[0] != translated_texts[0]
+        block_id = parsing_html_params[:block_id]
+
+        Card.delay.create_delayed_job(current_user.id, block_id, url,
+                                      css_original, css_translated)
+
+        flash.now[:notice] = 'Задача на парсинг сайта успешно поставлена в очередь.'
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.